domain-ee/ee-ep-merge-app/src/python_src/service/hoppy_service.py

```diff
@@ -3,12 +3,14 @@
 from config import EXCHANGES, QUEUES, REPLY_QUEUES, ClientName, config
 from hoppy.async_hoppy_client import AsyncHoppyClient, RetryableAsyncHoppyClient
 from hoppy.hoppy_properties import ExchangeProperties, QueueProperties
+from typing import Dict
 
 
 class HoppyService:
     clients: dict[ClientName, AsyncHoppyClient] = {}
 
     def __init__(self) -> None:
+        self.create_client(ClientName.DEAD_LETTER)
```

Comment:
This application does not need a client to the DLQ. This can be removed along with the logic in `create_client` and the queue and response-queue names in `config.py`.

Reply:
It's needed because it creates the dead letter queue for the queues that use it. If the dead letter queue doesn't exist when another queue tries to use it, the configuration is just dropped and the pipeline fails later down the line when bip-api tries to connect.